notebooks/swarm.ipynb

```diff
@@ -0,0 +1,1106 @@
+{
```

Comment:
<p>Can you add more info here about the Agents and the tools they have access to? I got a bit lost in the code cell to understand the tools. Something like this:</p><p> </p><ul><li>Triage Agent: handles general questions and directs to other agents. Tools: <strong>transfer_to_sales_agent</strong>, <strong>transfer_to_issues_and_repairs </strong>and <strong>escalate_to_human</strong></li><li>Sales Agent: tries to sell the product to the user, it can execute the order or redirect user back to the triage agent. Tools: <strong>xecute_order</strong>, <strong>transfer_back_to_triage</strong></li><li>Issues and Repairs Agent: supports to customer with their problems, it can look up item ID, execute refund or redirect user back to triage. Tools: <strong>look_up_item</strong>, <strong>execute_refund</strong>, <strong>transfer_back_to_triage</strong></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
good suggestions!